tests/functional.fmf

```diff
@@ -0,0 +1,37 @@
+summary:
+  Functional tests
+require:
+  - python3-pytest
+  - python3-specfile
+  - redhat-rpm-config
+  - rpmautospec-rpm-macros
+  - MUMPS-srpm-macros
+  - ansible-srpm-macros
+  - blis-srpm-macros
+  - efi-srpm-macros
+  - folly-srpm-macros
+  - fonts-srpm-macros
+  - fpc-srpm-macros
+  - ghc-srpm-macros
+  - gnat-srpm-macros
+  - go-srpm-macros
+  - kernel-srpm-macros
+  - lua-srpm-macros
+  - nbdkit-srpm-macros
+  - ocaml-srpm-macros
+  - openblas-srpm-macros
+  - package-notes-srpm-macros
+  - perl-srpm-macros
+  - pyproject-srpm-macros
+  - python-srpm-macros
+  - qt5-srpm-macros
+  - qt6-srpm-macros
+  - rust-srpm-macros
+  - typelib-srpm-macros
```

Comment:
Does the fmf accepts `*-srpm-macros`?

Reply:
Let me try, dnf seems to support it, so it could work.